smoke-tests/src/test/java/com/splunk/opentelemetry/TomcatSmokeTest.java

```diff
@@ -30,39 +33,63 @@
       new TomcatAttributes("7.0.107.0");
   public static final ExpectedServerAttributes TOMCAT8_SERVER_ATTRIBUTES =
       new TomcatAttributes("8.5.60.0");
+  public static final ExpectedServerAttributes TOMCAT8561_SERVER_ATTRIBUTES =
+      new TomcatAttributes("8.5.61.0");
```

Comment:
Why add this version?

Reply:
Removed. Initially this was because I used a download endpoint in Dockerfile building which only had the latest version, but since I changed that, it is no longer necessary.